### DISCUSSION TITLE
Problems building the storage-kit

### DISCUSSION THREAD

**Discussion by danimaraas** [walt-id/.github#74] [Help] [answered]
After cloning the waltid-storagekit repository, I failed to build the project. There is a Dockerfile in the project, but no docker-compose.yml or docker-compose.yaml. In other words, the third step of the "Installation and running the project" from https://docs.walt.id/v/storage-kit/getting-started/rest-apis doesn't seem to work.

**Comment by taminobaumann** [comment, marked as answer]
Hey @danimaraas, 

Thanks for reaching out! The storage kit landed in the backlog, as not many customers asked for it, which is why the current build is not working as well as the docker. As demand is increasing, we are looking into upgrading the code. However, I don't have a concrete timeline yet for when it will be running again.
